sempyro/hri_dcat/hri_dataset.py

```diff
@@ -36,53 +38,54 @@ class HRIDataset(RDFModel):
+        description="Relevant contact information for the cataloged resource. HRI mandatory",
         rdf_term=DCAT.contactPoint,
         rdf_type="uri")
 
     creator: List[Union[AnyHttpUrl, Agent]] = Field(
         description="The entity responsible for producing the resource. Resources of type foaf:Agent are "
-                    "recommended as values for this property.",
+                    "recommended as values for this property. HRI mandatory",
         rdf_term=DCTERMS.creator,
         rdf_type="uri")
 
     description: List[LiteralField] = Field(
-        description="A free-text account of the resource.",
+        description="A free-text account of the resource. HRI mandatory",
         rdf_term=DCTERMS.description,
         rdf_type="literal"
     )
     issued: Union[str, datetime, date, AwareDatetime, NaiveDatetime] = Field(
-        description="Date of formal issuance (e.g., publication) of the resource.",
+        description="Date of formal issuance (e.g., publication) of the resource. HRI mandatory",
         rdf_term=DCTERMS.issued,
         rdf_type="datetime_literal"
     )
     identifier: Union[str, LiteralField] = Field(
-        description="A unique identifier of the resource being described or cataloged.",
+        description="A unique identifier of the resource being described or cataloged. HRI mandatory",
         rdf_term=DCTERMS.identifier,
-        rdf_type="xsd:string")
+        rdf_type="xsd:string"
+    )
     modified: Union[str, date, AwareDatetime, NaiveDatetime] = Field(
-        description="Most recent date on which the resource was changed, updated or modified.",
+        description="Most recent date on which the resource was changed, updated or modified. HRI mandatory",
         rdf_term=DCTERMS.modified,
         rdf_type="datetime_literal"
     )
     publisher: List[Union[AnyHttpUrl, Agent]] = Field(
-        description="The entity responsible for making the resource available.",
+        description="The entity responsible for making the resource available. HRI mandatory",
         rdf_term=DCTERMS.publisher,
         rdf_type="uri"
     )
     theme: List[AnyHttpUrl] = Field(
-        description="A main category of the resource. A resource can have multiple themes.",
+        description="A main category of the resource. A resource can have multiple themes. HRI mandatory",
         rdf_term=DCAT.themeTaxonomy,
         rdf_type="uri"
     )
     title: List[LiteralField] = Field(
-        description="A name given to the resource.",
+        description="A name given to the resource. HRI mandatory",
         rdf_term=DCTERMS.title,
         rdf_type="rdfs_literal"
     )
     type: List[AnyHttpUrl] = Field(
-        description="The nature or genre of the resource.",
+        description="The nature or genre of the resource. HRI mandatory",
         rdf_term=DCTERMS.type,
         rdf_type="uri")
     license: AnyHttpUrl = Field(
```

Comment:
Could be removed, it is already in parent class?

Reply:
my preference would be to keep it so it is explicit what are HRI mandatory/recommended fields